packages/mds-db/migration.ts

```diff
@@ -125,13 +131,15 @@ async function doMigration(
             schema.TABLE_COLUMNS.migrations
           )}) VALUES ('${migration}', ${now()})`
         )
+        process.env.PG_DEBUG = 'true'
```

Comment:
What's the purpose of doing this? It just takes the point out of having environment variables.

Reply:
Just ensuring that the SQL commands associated with migrations are always logged. It does not affect the way that the environment variable is used elsewhere. It returns it to its previous value after the migrations are run.

Reply:
Perhaps we could remove the `PG_DEBUG` check in `logSql` altogether then?

Reply:
No ... We still want to be able to control PG_DEBUG to suppress excessive logging during normal operations. Migrations are a special case where we always want debugging to be enabled. They run infrequently and it's important to be able to monitor what DDL commands were attempted.

Reply:
Ok, that makes sense.